tests/zfs-tests/tests/perf/fio/random_readwrite_fixed.fio

```diff
@@ -30,7 +31,10 @@ ioengine=psync
 sync=${SYNC_TYPE}
 direct=${DIRECT}
 numjobs=${NUMJOBS}
+randseed=1234
```

Comment:
Similar to the other values set by environment variables (RUNTIME, FILESIZE, NUMJOBS, etc) it would be nice to be able to override `randseed` with `PERF_RANDSEED`.  Having it default to a constant seed for consistently seems reasonable.  The other global tuning are set at the top of `tests/zfs-tests/tests/perf/perf.shlib`.

Reply:
If you can come up with a reason to have a different seed, thats fine.
But I cant really think of any reason to use a different random-number seed for one test and not for another.

Reply:
Fixed it anyway, following commit should be about what you requested.